meta-resin-common/recipes-core/systemd/systemd/coredump.conf

```diff
@@ -0,0 +1,2 @@
+[Coredump]
+Storage=none
```

Comment:
Just to clarify: what I propose is to have the coredump stored in journal for development and not available completely in production.